spec/e2e/migration_spec.rb

```diff
@@ -13,7 +13,7 @@ class ThirdRelClass; end
       class User
         include Neo4j::ActiveNode
         property :name
-        has_many :out, :songs, model_class: :Song, type: 'songs'
+        has_many :out, :songs, model_class: :'MigrationSpecs::Song', type: 'songs'
```

Comment:
I actually did some refactoring of this just a couple of days ago so that the `MigrationSpecs` module isn't needed anymore.  I just merged [the PR](https://github.com/neo4jrb/neo4j/pull/1161) into master so you can try merging it into you branch if you like.  Otherwise we'll take care of it when this PR is done